src/main/java/frc/robot/constants/RobotComponents.java

```diff
@@ -95,4 +96,12 @@
                 CAN.Swerve.REAR_RIGHT_DRIVE_MOTOR_ID, DRIVE_MOTOR_CONFIG.withPID(
                 LOCAL_SWERVE_MODULES.rearRightModuleConstants.driveCoefs));
     }
+
+    protected static class ShooterComponents {
+        // TODO: Set correct MotorConfig
+        public static final MotorConfig MOTOR_CONFIG = new MotorConfig().withOpenLoopRampRate(3);
+        public static final TrigonTalonFX RIGHT_MOTOR = new TrigonTalonFX(CAN.Shooter.RIGHT_MOTOR_ID, MOTOR_CONFIG);
+        public static final TrigonTalonFX LEFT_MOTOR = new TrigonTalonFX(
```

Comment:
left above right just because of the alphabet

Reply:
refer to above

Reply:
fix